kit/Kit.cpp

```diff
@@ -300,12 +299,6 @@ class LinkOrCopy
             break;
         case FTW_D:
             {
-                struct stat st;
```

Comment:
That looks like a functional change to the casual observer; that's very unexpected in something that says it just addresses renames.

Reply:
We are given the file's stat. Why are we doing another stat call? No functional change, just using the existing stat results.